data_store/cli.py

```diff
@@ -40,6 +41,7 @@ def seed_reference():
         with current_app.app_context():
             seed_geospatial_dim_table()
             seed_fund_table()
+            load_example_data()
```

Comment:
I don't think this should be part of the `seed-ref` command - it's not reference data.

We could add/restore a separate command for loading this example data, and then add that to the docker runner startup.